toolkit/Makefile

```diff
@@ -139,9 +139,36 @@ SRPM_URL_LIST      ?= https://packages.microsoft.com/cbl-mariner/$(RELEASE_MAJOR
 # Variable prerequisite tracking
 include $(SCRIPTS_DIR)/utils.mk
 
-######## REMAINING BUILD FLAGS ########
+######## DAILY BUILD AND PREVIEW REPO SUPPORT ########
```

Comment:
Considering the comment, do you think it's time to create another `*.mk` file and move this logic there?;)

Reply:
I was debating it actually... I can do that.